doc/ornate.conf

```diff
@@ -91,8 +91,12 @@ extension.externalLinks {
     uri = "https://issues.scala-lang.org/browse/SI-[all]"
     text = "SI-[all]"
   }
-  samplezip.uri = "https://example.lightbend.com/v1/download/[all]-"${shortVersion}
-  samplerepo.uri = "https://github.com/slick/slick/tree/"${shortVersion}"/samples/[all]"
+  sampleCodeZip{
```

Comment:
Curious why you changed to the curly brace syntax?

Reply:
Thanks for reaching out I was experimenting with the ornate as I am new to it, that's the reason I have kept the PR in the draft. 
The curly braces were just the part of it actually `sampleCodeZip.uri ` does the same, there are some changes since the last commit which I have to push this weekend

Reply:
Ok, I set the issue assignee to you to indicate that it's awaiting your changes. When you push them you can unassign yourself or assign me to indicate that you're done.

Reply:
@initvik hi, what's the status of this?